.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false
```

Comment:
This should be a linter thing, not vscode. I [made a PR](https://github.com/CenterForOpenScience/ember-osf-web/pull/256) enforcing it.

Reply:
It just makes it so that VS Code will apply that setting on format and will comply with that linting rule. Otherwise, the auto format feature will put the space in and then will have to be corrected

Reply:
Right, but the linting rule wasn't enabled before. I'd hope that VS Code can use eslint, rather than requiring us to commit/maintain editor-specific configs.

Reply:
If you don't use VS Code, you won't be affected

Reply:
But it should be enforced for everyone, regardless of editor choice. That's what linters are for.  Once #256 is merged, this file shouldn't be necessary.

Reply:
They're not mutually exclusive